### PR TITLE
add redis 6

### PR DESCRIPTION
**Pull request by coreypurcell**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,12 +11,14 @@ env:
   - STACK=heroku-18 REDIS_VERSION=3
```
Comment (edmorley) [inline .travis.yml:12]: I think we should also test on Heroku-16, given it's still supported.

```suggestion
  - STACK=heroku-16 REDIS_VERSION=6
  - STACK=heroku-18 REDIS_VERSION=3
```

**Review by edmorley — APPROVED** [review]
Thank you for the PR :-)

**Comment by coreypurcell** [comment]
@edmorley is there a way to test this. Or is the travis build enough?

**Comment by edmorley** [comment]
@coreypurcell The Travis build should be enough :-)

(Feel free to merge whenever)

**Pull request merged**